src/__stories__/Skeleton.stories.tsx

```diff
@@ -394,3 +396,23 @@ export const RegressionTest133 = () => (
         </div>
     </div>
 )
+
+const FourthDiv = () => <div>Fourth Div</div>
+export const UseSkeletonHook: React.FC = () => {
```

Comment:
Could we move this to its own `.stories.tsx` file? Since we have a ton of stories in this file.